### PR TITLE
Add node uptime func

### PR DESCRIPTION
**Pull request by adizaimi**
add comdb2_node_uptime() and comdb2_node_starttime() sql functions

### CODE REVIEW

**Comment by riverszhang89** [comment]
`ps` can tell us these.

**Comment by nirbhayc** [comment]
Won't work with `nowatch`.

**Comment by adizaimi** [comment]
While ps can show you that and more info, you'd need to login onto the box to get that and, besides the inconvenience of doing that, you might not even have shell access onto a box.

**Comment by adizaimi** [comment]
It does indeed work even with nowatch because nowatch does not disable the thread from running, but it skips over doing work if gbl_nowatch is set.

**Comment by riverszhang89** [comment]
Fair. But please avoid using node. We even changed comdb2_node() to comdb2_host().

**Comment by adizaimi** [comment]
ok, no need for node in the name. thanks.

File: sqlite/ext/misc/completion.c [hunk 1]
```diff
@@ -338,7 +338,7 @@ static int completionNext(sqlite3_vtab_cursor *cur){
         break;
       }
       case COMPLETION_FUNCTIONS: {
-        static char *cfuncs[] = {"comdb2_version()", "table_version()", "partition_info()", "comdb2_host()", "comdb2_port()", "comdb2_dbname()", "comdb2_prevquerycost()", "sys.cmd.send()"};
+        static char *cfuncs[] = {"comdb2_version()", "table_version()", "partition_info()", "comdb2_host()", "comdb2_port()", "comdb2_dbname()", "comdb2_prevquerycost()", "comdb2_uptime()", "comdb2_starttime()", "sys.cmd.send()"};
```
Comment (nirbhayc) [inline sqlite/ext/misc/completion.c]: Please break this into multiple lines.

**Comment by nirbhayc** [comment]
@adizaimi: Please also update the docs.

**Pull request merged**